### ISSUE TITLE
Video / User Manual

### ISSUE THREAD

**Issue by GoodbyeKittyy** [T-wcs/RansomPy-Covid19#3]
Hi there,

Is there any video, or user manual that I can refer to run this example? I am finding related projects to showcase in a demonstration :)

**Comment by T-wcs** [comment]
No, I haven't taken the time to make any videos, but if you have any questions, don't hesitate to ask me and I'll get back to you.
